extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -129,18 +136,41 @@ class AmpStickyAd extends AMP.BaseElement {
         const borderBottom = this.element./*OK*/offsetHeight;
         this.viewport_.updatePaddingBottom(borderBottom);
         this.addCloseButton_();
-        timer.delay(() => {
-          // Unfortunately we don't really have a good way to measure how long it
-          // takes to load an ad, so we'll just pretend it takes 1 second for
-          // now.
-          this.vsync_.mutate(() => {
-            this.element.classList.add('amp-sticky-ad-loaded');
-          });
-        }, 1000);
+      });
+      // Get Ad promise for ad render-start
+      this.adRenderStartPromise_ = this.ad_.implementation_.renderStartPromise;
```

Comment:
The `implementation_` will not be available here in PROD binary given the private property renaming.